Config.h

```diff
@@ -6,6 +6,72 @@
 
 constexpr auto KEYMOD_NONE = 0;
 
+constexpr bool DEFAULT_FULLSCREEN_MODE = false;
+constexpr int DEFAULT_RESOLUTION_WIDTH = 800;
+constexpr int DEFAULT_RESOLUTION_HEIGHT = 600;
+constexpr int DEFAULT_SIZE = /*20*/ 70;
+constexpr int DEFAULT_SIZE_Z = /*6*/ 4;
+
+class Config {
+    // constexpr auto GFXMODE = GFX_AUTODETECT_WINDOWED;
```

Comment:
leftover?

Reply:
yeah it's a relic of days gone by, if someone wants to find it there's always git history